src/main/java/com/createment/springone/RouteController.java

```diff
@@ -12,7 +12,7 @@
 @Controller
 public class RouteController {
 
-    final String DB_HOST = "jdbc:mysql://localhost:3306/bookstore";
+    final String DB_HOST = "jdbc:mysql://192.168.1.191:3306/bookstore";
```

Comment:
Why do you do this?

Reply:
my docker is running on the laptop. im on my main PC. i NEED to use the internal ip to access the laptop